tests/phpunit/Console/E2ETest.php

```diff
@@ -140,12 +140,12 @@ public function test_it_runs_on_itself(): void
             $this->markTestSkipped(implode("\n", [
                 'Refusing to run Infection on itself with no memory limit set: it is dangerous.',
                 'To run this test with a memory limit set please use:',
-                'php -dmemory_limit=128M vendor/bin/phpunit --group=large',
+                'php -dmemory_limit=512M vendor/bin/phpunit --group=large',
             ]));
         }
 
         $output = $this->runInfection(self::EXPECT_SUCCESS, [
-            '--test-framework-options="--exclude-group=' . self::EXCLUDED_GROUP . '"',
+            sprintf('--test-framework-options=--exclude-group=%s', self::EXCLUDED_GROUP),
```

Comment:
I'm guessing, but probably quotes affects somehow the build. It's now written in a different way

Reply:
It didn't help  :see_no_evil:

Reply:
not many cases left to revert :) next is `--no-progress`?

Reply:
It's more line something *isn't added* here.